primitives/sidechain/src/traits/mod.rs

```diff
@@ -21,14 +21,13 @@
 //! some generic structs.
 
 use codec::{Decode, Encode};
-use core::hash::Hash;
-use sp_core::{blake2_256, Pair, Public, H256};
-use sp_runtime::traits::Member;
+use sp_core::{crypto::Public, H256};
```

Comment:
@gaudenzkessler This one is mean. Public is available with `no_std` but just not re-exported at top level.